test/tools/ossfuzz/protoToYul.h

```diff
@@ -314,9 +316,13 @@ class ProtoConverter
 	std::vector<std::string const*> m_currentGlobalVars;
 	/// Functions in current scope
 	std::vector<std::vector<std::string>> m_scopeFuncs;
-	/// Variables
+	/// Global variables
 	std::vector<std::vector<std::string>> m_globalVars;
-	/// Functions
+	/// Variables declared in for loop init block that is in global scope
+	std::vector<std::vector<std::string>> m_globalForLoopInitVars;
+	/// Variables declared in for loop init block that is in function scope
+	std::vector<std::vector<std::vector<std::string>>> m_funcForLoopInitVars;
```

Comment:
Two data structures for tracking variables declared per for-init block. Global data structure is used to track for-init vars in for loops in global scope. It is 2D vector of varref strings because we need one per scope-for pair.

Function data structure tracks for-init vars per function. It is 3D because of function is an added dimension.